AirportPanel/Program.cs

```diff
@@ -0,0 +1,490 @@
+                                {
+                                    ++j;
+                                }
+                            }
+                            if (j <= 19) FlightList[j] = AddFlight();
+                            break;
+                        }
+                    case 'q':
+                    case 'Q': // закрытие приложения
+                        {
+                            Environment.Exit(0);
+                            break;
+                        }
+                    default:
+                        break;
+                }
+            }
+
+        }
+
+        public static Flight AddFlight()
+        {
+            string s;
+            Console.Clear();
+            Console.BackgroundColor = ConsoleColor.Red;
+            Console.WriteLine(CentrStr("Add flight info:", 164));
+            Console.BackgroundColor = ConsoleColor.Black;
+            Console.ForegroundColor = ConsoleColor.Blue;
+            Console.WriteLine(new string('-', 164));
+            Console.WriteLine("| Input type of flight (In/Out)                            |");
+            Console.WriteLine("| Input number of flight (1 to 999)                        |");
+            Console.WriteLine("| Input date/time of flight (dd.mm.yyyy hh:mm)             |");
+            Console.WriteLine("| Input target city (help at the bottom of the screen)     |");
+            Console.WriteLine("| Input airline company (help at the bottom of the screen) |");
+            s = $"Input terminal code ({(TerminalName)1}/";
+            for (int i = 2; i <= Enum.GetNames(typeof(TerminalName)).Length-1; i++)
+            {
+                s += (CityName)i + "/" + i + " ";
+            }
+            s += $"{(TerminalName)Enum.GetNames(typeof(TerminalName)).Length})";
+            Console.WriteLine($"| {s}" + new string(' ', 56 - s.Length) + " |");
+            Console.WriteLine("| Input gate in the terminal (1 to 999)                    |");
+            Console.WriteLine("| Input flight status (help at the bottom of the screen)   |");
+            Console.ForegroundColor = ConsoleColor.White;
+
+            Flight flight = new Flight
+            {
+
+                InOut = InputInOut(61, 2),
+                Number = InputNumberGate(61, 3),
+                DTFlight = InputDT(61, 4),
+                City = InputCity(61, 5),
+                Airline = InputAirline(61, 6),
+                Terminal = InputTerminal(61, 7),
+                Gate = InputNumberGate(61, 8),
+                Status = InputStatus(61, 9),
+            };
+
+            Console.BackgroundColor = ConsoleColor.Black;
+            Console.SetCursorPosition(0, 40);
+            Console.Write(new string(' ', 164));
+            Console.SetCursorPosition(0, 40);
+            Console.WriteLine("MENU: 1-ViewList 2-AddFlight 3-DeliteFlight 4-EditFlight 5-FindFlight 6-SpecialFunction Q-Quit");
+
+            return flight;
+        }
+        public static void TableHead()
```

Comment:
Remember about code convension.

Reply:
I agree, it was necessary to work through constants (40 and 160) ...,

automate calculation 61 (Airline = InputAirline (61, 6),
+ Terminal = InputTerminal (61, 7),
+ Gate = InputNumberGate (61, 8),
+ Status = InputStatus (61, 9), is through the structure of the header and its method for calculating the maximum length?